app/services/api/append_metadata.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Api
+  class AppendMetadata
+    def initialize(dataset)
+      @dataset = dataset
+    end
+
+    def call
+      {
+        set_name => dataset,
```

Comment:
should this always just be "items" or do we want to label it relative to the dataset?

Reply:
I think `data` is probably the right name for this. (I know we don't need to follow JSON-API but in the absence of any other prior-art it makes sense to be consistent as we know that the consumers of this API will already be used to those conventions). In any case I think making it specific to the type of thing makes things more complicated for the client so I'm in favour of a fixed name.

Reply:
agreed - I'll change it to `data`